### DISCUSSION TITLE
Pulling new events

### DISCUSSION THREAD

**Discussion by Sru3l** [MISP/MISP#8253] [Q&A]
Hi team, asking here because i cannot find the solution - 
Is it possible to pull new events based on the timestamp/time of server pulling the events ?
I was not able to achieve this with cronjobs using UPDATE|FULL - to clarify, when using UPDATE i found it only pulled updates to local events that are already there and FULL would pull all the events from the given server.

What i want to do is **pulling events for the day and not older events.** In some cases i would only want ''from now'' and not behind that so i can start to pull events daily and making sure i do not get old events and populate my DB.

Any ideas if it is possible?

**Comment by iglocska** [comment]
Sure thing! You can't directly do it from the cron job / command line, but when configuring the sync connection you can add "pull filters". Just edit the sync connection you wish to apply such filters on, hit the pull filters button and at the bottom you can add custom parameters such as timestamp or publish_timestamp.
